docs/buildah-build.1.md

```diff
@@ -96,7 +96,37 @@ The value of `[name]` is matched with the following priority order:
 
 **--cache-from**
 
-Images to utilise as potential cache sources. Buildah does not currently support --cache-from so this is a NOOP.
+Repository to utilise as potential cache source. When specified buildah will try to look for
+cache images in specified repository and will attempt to pull cache images instead of actually
+executing and computing STEPS. Buildah will only attempt to pull cache images if they are considered
+as valid cache hits otherwise not.
+
+Use `--cache-to` to populate a remote repository with cache content
+
+Example
+
+```bash
+# populate a cache and also consult it
+buildah build -t test --layers --cache-to registry/myrepo/cache --cache-from registry/myrepo/cache .
+```
+
+Note: option is ignored unless `--layers` is specified.
```

Comment:
which option is ignored?  use used `-t1, `--cache-to`, and `--cache-from`

Reply:
This is the section about `--cache-from`, but I don't mind if it's named on this line.

Reply:
fixed.